src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -623,7 +620,7 @@ TypeOrNamespaceUsageInfo GetTypeOrNamespaceUsageInfo()
                 ? TypeOrNamespaceUsageInfo.Qualified
                 : TypeOrNamespaceUsageInfo.None;
 
-            if (semanticFacts.IsNamespaceDeclarationNameContext(semanticModel, node.SpanStart, cancellationToken))
+            if (isInNamespaceNameContext)
```

Comment:
this was a very inefficient call.  We already had the node.  But we used it to just get its position, to do a bunch of expensive tree work (and we do this for *all* matches we find across all files).